tests/test_logging.py

```diff
@@ -627,6 +635,56 @@ def test_handler_knows_how_large_logs_are(self):
         assert handler._get_payload_size(dict_log) == log_size
 
 
+class TestWorkerLogging:
```

Comment:
Can you add a test asserting that worker logs are received by the `APILogWorker` as expected?